_inc/client/security/backups-scan.jsx

```diff
@@ -12,37 +14,127 @@ import ExternalLink from 'components/external-link';
+	getVaultPressData as _getVaultPressData
+} from 'state/at-a-glance';
+import { isFetchingSiteData } from 'state/site';
+import QueryVaultPressData from 'components/data/query-vaultpress-data';
 
-export const BackupsScan = moduleSettingsForm(
-	React.createClass( {
-
-		toggleModule( name, value ) {
-			this.props.updateFormStateOptionValue( name, !value );
-		},
-
-		render() {
-			return (
-				<SettingsCard
-					feature={ FEATURE_SECURITY_SCANNING_JETPACK }
-					{ ...this.props }
-					header={ __( 'Backups and security scanning', { context: 'Settings header' } ) }
-					hideButton>
-					<SettingsGroup disableInDevMode module={ { module: 'backups' } } support="https://vaultpress.com/jetpack/">
-						<p>
-							{
-								__( 'Your site is backed up and threat-free.' )
-							}
-						</p>
-						{
-							! this.props.isUnavailableInDevMode( 'backups' ) && (
-								<span>
-									<ExternalLink className="jp-module-settings__external-link" href="https://dashboard.vaultpress.com/" >{ __( 'Configure your Security Scans' ) }</ExternalLink>
-								</span>
-							)
-						}
-					</SettingsGroup>
-				</SettingsCard>
-			);
+const BackupsScan = React.createClass( {
+	toggleModule( name, value ) {
+		this.props.updateFormStateOptionValue( name, !value );
+	},
+
+	getNotice() {
+		const vpData = this.props.vaultPressData,
+			scanEnabled = get( vpData, [ 'data', 'features', 'security' ], false ),
+			hasPremium = /jetpack_premium*/.test( this.props.sitePlan.product_slug ),
+			hasBusiness = /jetpack_business*/.test( this.props.sitePlan.product_slug );
+
+		let status,
+			text,
+			link,
+			icon = '';
+
+		if ( this.props.isModuleActivated( 'vaultpress' ) ) {
+			if ( vpData === 'N/A' ) {
+				status = 'is-info';
+				text = __( 'Loading…' );
+				link = false;
+			} else if ( scanEnabled ) {
+				if ( 0 !== this.props.scanThreats ) {
+					status = 'is-error';
+					text = __( 'Threats found!', { context: 'A message about security threats found.' } );
+					link = 'https://dashboard.vaultpress.com/';
+				} else if ( vpData.code === 'success' ) {
+					status = 'is-success';
+					text = __( 'All clean!', { context: 'A message about no security threats found.' } );
+					link = false;
+					icon = 'checkmark';
+				}
+			} else {
+				status = 'no-pro-uninstalled-or-inactive';
+				text = __( 'Inactive', { context: 'A message about the security plugin not activated yet.' } );
+				icon = 'notice';
+			}
+		} else if ( hasPremium || hasBusiness ) {
+			status = 'is-warning';
+			text = __( 'Inactive', { context: 'A message about the security plugin not activated yet.' } );
+			link = 'https://wordpress.com/plugins/vaultpress';
+			icon = 'notice';
+		} else if ( this.props.fetchingSiteData ) {
+			status = 'is-info';
+			text = __( 'Loading…' );
+			link = false;
+		} else {
+			status = 'is-warning';
+			text = __( 'Upgrade', { context: 'Caption for a button to purchase a paid feature.' } );
```

Comment:
So what I meant to say is that you're right @dereksmart . This condition branch is not going to be needed because we're going to be hiding the card in this case.